### PR TITLE
Added graph functions for EPV report

### PR DESCRIPTION
**Pull request by yzainee**

### CODE REVIEW

**Comment by samuzzal-choudhury** [comment]
@yzainee could you please correct the build failures?

**Comment by yzainee** [comment]
@samuzzal-choudhury For some reason its not picking the latest files to do a build. Its picking up some old files and throwing error

File: f8a_report/graph_report_generator.py [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+        if gremlin_response is not None:
+            result_data += get_response_data(gremlin_response, [{0: 0}])
+        else:
+            _logger.error("Error while trying to fetch data from graph. "
+                          "Expected response, got None...Query->", query)
+
+    return result_data
+
+
+"""
```
Comment (samuzzal-choudhury) [inline f8a_report/graph_report_generator.py]: Please remove this commented out block. This need not go to the master branch.

File: runtests.sh [hunk 2]
```diff
@@ -66,6 +66,8 @@ export AWS_S3_REGION="not-set"
 prepare_venv
 pip3 install -r requirements.txt
 pip3 install -r tests/requirements.txt
+pip3 install git+https://github.com/fabric8-analytics/fabric8-analytics-utils.git@d7aaccf
```
Comment (samuzzal-choudhury) [inline runtests.sh]: @yzainee for better maintainability, please set the commit hash in a variable in the script and use that variable with the `pip3 install` here.

File: f8a_report/graph_report_generator.py [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+GREMLIN_SERVER_URL_REST = "http://gremlin-yzainee-fabric8-analytics.devtools-dev.ext.devshift.net"
+"""
+GREMLIN_SERVER_URL_REST = "http://{host}:{port}".format(
+    host=os.environ.get("BAYESIAN_GREMLIN_HTTP_SERVICE_HOST", "localhost"),
+    port=os.environ.get("BAYESIAN_GREMLIN_HTTP_SERVICE_PORT", "8182")) """
+
+GREMLIN_QUERY_SIZE = os.getenv('GREMLIN_QUERY_SIZE', 50)
+
+"""
+TODO: WIP
```
Comment (samuzzal-choudhury) [inline f8a_report/graph_report_generator.py]: Let us not push any WIP code to the master even if those are commented out :-)

File: f8a_report/graph_report_generator.py (hunk 1 above)
Comment (yzainee) [reply f8a_report/graph_report_generator.py]: removed

File: f8a_report/graph_report_generator.py (hunk 3 above)
Comment (yzainee) [reply f8a_report/graph_report_generator.py]: removed

File: runtests.sh (hunk 2 above)
Comment (yzainee) [reply runtests.sh]: not using the commit hash as its expected to use the latest version comparator code everytime. let me know if you think otherwise

Comment (samuzzal-choudhury) [reply runtests.sh]: @yzainee my comment was for `git+https://github.com/fabric8-analytics/fabric8-analytics-utils.git`. Please keep the commit hash for utils in sync with the one used in the Dockerfile and also, although nitpick, please parameterize the commit hash for `fabric8-analytics-utils.git` in `runtests.sh`

**Pull request merged**